README.md

```diff
@@ -33,6 +34,8 @@ $ svgdigitizer digitize test/data/Ni111_NaOH_Beden1985_Fig2c.svg
 [creates test/data/Ni111_NaOH_Beden1985_Fig2c.csv]
 $ svgdigitizer paginate <pdf file>
 [paginates the pdf file]
+$ svgdigitizer cv test/data/Ni111_NaOH_Beden1985_Fig2c.svg
+[creates test/data/Ni111_NaOH_Beden1985_Fig2c.csv]

```

Comment:
We could distinguish the two `.csv` outputs (this one and the one of `digitize`) by saying something like `creates ...csv with axes x,y,z`?